python/dask_cudf/dask_cudf/core.py

```diff
@@ -434,6 +419,80 @@ class Index(Series, dd.core.Index):
     _partition_type = cudf.Index
 
 
+def _naive_var(ddf, meta, skipna, ddof, split_every, out):
+    num = ddf._get_numeric_data()
+    x = 1.0 * num.sum(skipna=skipna, split_every=split_every)
+    x2 = 1.0 * (num ** 2).sum(skipna=skipna, split_every=split_every)
+    n = num.count(split_every=split_every)
+    name = ddf._token_prefix + "var"
+    result = map_partitions(
+        var_aggregate, x2, x, n, token=name, meta=meta, ddof=ddof
+    )
+    if isinstance(ddf, DataFrame):
+        result.divisions = (min(ddf.columns), max(ddf.columns))
+    return handle_out(out, result)
+
+
+def _parallel_var(ddf, meta, skipna, split_every, out):
+    def _local_var(x, skipna):
+        n = len(x)
+        # TODO: x.sum()/n seems to be faster than x.mean()
+        # on Quadro RTX 8000 - Need to compare on V/A100
+        avg = x.mean(skipna=skipna)
```

Comment:
Interesting result:  I found that the new approach is slightly slower on my local RTX 8000-based machine **unless** I use `x.sum()/n` in place of `x.mean()`.  I plan to get an environment set up on a V100-based machine soon.  I will see if I get different behaviour on datacenter-grade hardware and update here.

Reply:
Should we hold off on merging until you report back on this?

Reply:
Interesting! And very surprising. If I can help dig into that one at all, please feel free to ping me.

Reply:
>Should we hold off on merging until you report back on this?

Not sure - I should have an update in the next few hours (just set up a new env, and building cudf now).  Should we remove the "naive" path entirely if the new approach has good performance?  Either way,  if `mean` is still slower than `sum/n` I will certainly welcome help from @wphicks :)

Reply:
We may want to be careful here. `x.mean()` by default is not the same as `x.sum()/len(x)` if x has nulls and skipna is True (default)

Reply:
Also Nick's point above may explain some of the performance difference (handling nulls vs. not)

Reply:
>We may want to be careful here. x.mean() by default is not the same as x.sum()/len(x) if x has nulls and skipna is True (default)

Ah - The case I was testing did not have nulls, but this is a good point.

Reply:
I can confirm that the "sum/n is faster than mean" statement was only true because `n` was a scalar value.  If `skipna=True`, we actually need the full `count` for `n`, because there may be null values.  In the `skipna=True` case, `mean` is faster (as expected).

Currently, the new `var` approach seems to be a bit slower than the original, so it may make sense to keep the `naive=True` option for now.